examples/ResNet50RetinaNet.ipynb

```diff
@@ -153,17 +153,17 @@
     "boxes /= scale\n",
     "\n",
     "# visualize detections\n",
-    "for idx, (label, score) in enumerate(zip(predicted_labels, scores)):\n",
-    "    if score < 0.5:\n",
-    "        continue\n",
-    "        \n",
-    "    color = label_color(label)\n",
+    "for i in np.where(scores > 0.5)[0]:\n",
```

Comment:
It would be easier to loop over `nms_classification` directly:
```python
for i, label in np.transpose(np.where(nms_classification > 0.5)):
    b = boxes[i, :].astype(int)
    ...
```

Then you can delete some of the variables as well (scores and predicted_labels).

Reply:
Ok, I will change, but I kept in that way for those which are learning the code (like me) make a little bit easier to understand.
Thank you.